### PR TITLE
accessories/glasses 

### PR DESCRIPTION
**Pull request by coleww**
Adds some basic logic for tossing accessories onto a cat, and glasses too!

accessories can go in `/parts` and just need to be added to the list in `lib/accessorize.js`

some cute lil example boogers:
screen shot 2016-02-02 at 10 57 31 am
screen shot 2016-02-02 at 10 57 24 am
screen shot 2016-02-02 at 10 57 18 am
screen shot 2016-02-02 at 10 57 10 am

### CODE REVIEW

**Comment by beaugunderson** [comment]
omg yes yes yes these are adorable!!! :heart_eyes_cat:

in the screenshots it looks like the left side has the tinted rectangle drawn over the frame, should be drawn underneath (maybe it's already fixed in the code?)

also, if you specify `lineWidth` it needs to be multiplied by `options.scaleFactor` (or else the frames are too thick in the grid view since each cat is smaller)

node does have string interpretation now too btw, it looks like ``string ${stuff[0]}`` :sparkles: (not necessary for this PR but i am excited that we can use it now!)

File: parts/glasses.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  // the radius from center of eye to rim of glasses for each axis
+  var xRad = options.eyeSize * _.random(2.5, 4)
+  var yRad = options.eyeSize * _.random(1.75, 2.5)
+  var isWearingSunglassesAtNight = _.random(0, 1) < 0.5
+  var glassesTintLevel = _.random(0, 0.5)
+  var glassesTint = _.sample([[0, 0, 0], [22, 55, 33], [40, 80, 60], [255, 105, 180], [25, 50, 13], [239, 89, 123], [243, 243, 21], [139, 69, 19], [10, 15, 5]])
+
+  // does node have string interps yet? no? ok, fine, TODO make this less awful.
+  ctx.fillStyle = "rgba(" + glassesTint[0] + ", " + glassesTint[1] + ", " + glassesTint[2] + ", " + glassesTintLevel + ")";
+  ctx.lineWidth = rimSize;
```
Comment (beaugunderson) [inline parts/glasses.js]: multiply by `options.scaleFactor` here so frames aren't too thick in grid mode

**Comment by coleww** [comment]
:pray: updated all the things :sparkles:

**Comment by beaugunderson** [comment]
merging all the things!!! :sparkles:

**Pull request merged**